tools/arcilator/arcilator.cpp

```diff
@@ -375,6 +414,36 @@ static LogicalResult processBuffer(
   if (failed(pmLlvm.run(module.get())))
     return failure();
 
+#ifdef ARCILATOR_ENABLE_JIT
+  // Handle JIT execution.
+  if (!runSimulation.empty()) {
+    mlir::ExecutionEngineOptions engineOptions;
```

Comment:
It’s simply because I don’t really know what that is. I figured the optimization option set to agressive would be enough but maybe I am missing parts of it. I’ll look into this tomorrow.